client/components/NewMessage.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+import people from '../people.js'
+
+const styles = {
+  borderRadius: '150px',
+  background: 'white',
+  width: '300px',
+  height: '300px'
+}
+
+class NewMessage extends React.Component {
+  constructor () {
+    super()
+    this.state = {
+      sender: null,
+      recipient: null,
+      senderPhoto: null,
+      recipientPhoto: null
+    }
+    this.handleChange = this.handleChange.bind(this)
+    this.handleGetPhoto = this.handleGetPhoto.bind(this)
+  }
+  handleGetPhoto (action, name) {
+    if (action === 'sender') {
+      people.map((person) => {
+        if (person.name === name) {
+          this.setState({senderPhoto: person.photo})
+        }
+      })
+    } else if (action === 'recipient') {
+      people.map((person) => {
+        if (person.name === name) {
+          this.setState({recipientPhoto: person.photo})
+        }
+      })
+    }
+  }
+
+  handleChange (e) {
+    const target = e.target
+    const name = target.value
+    const action = target.name
```

Comment:
You could reduce these 3 lines to 1 if you deconstructed `name` and `value` from `e.target`.